packages/cozy-procedures/src/components/EndModal.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
+import { withClient } from 'cozy-client'
+import {
+  Modal,
+  ModalContent,
+  ModalFooter,
+  Title,
+  translate,
+  Button,
+  ButtonLink
+} from 'cozy-ui/transpiled/react/'
+import AppLinker, {
+  generateUniversalLink
+} from 'cozy-ui/transpiled/react/AppLinker'
+import IllustrationOK from '../assets/ok.svg'
+import IllustrationError from '../assets/error.svg'
+
+class EndModal extends React.Component {
+  exitProcedure = () => {
+    this.props.router.push('/')
+  }
+
+  render() {
+    const { t, isSuccessful, onClose, client, folderId } = this.props
+    const translationKey = isSuccessful ? 'success' : 'error'
+    const cozyUrl = client.getStackClient().uri
+
+    const slug = 'drive'
+    const path = `#/folder/${folderId}`
+
+    const fallbackUrl = generateUniversalLink({
+      slug,
+      cozyUrl,
+      nativePath: path
+    })
+
+    return (
+      <Modal
+        secondaryAction={isSuccessful ? this.exitProcedure : onClose}
+        mobileFullscreen
+      >
+        <ModalContent>
+          <img
+            src={isSuccessful ? IllustrationOK : IllustrationError}
+            className="u-db u-w-4 u-h-4 u-mh-auto u-mb-1"
+          />
+          <Title className="u-mb-2 u-ta-center">
+            {t(`${translationKey}.title`)}
+          </Title>
+          <p className="u-mb-2 u-ta-center">{t(`${translationKey}.text`)}</p>
```

Comment:
Can we remove the translationKey var and put the ternary here? Since in the following line we already know if we are sucessful or not, we can write the right label directly. It'll be easier to find and replace. I'm not a big fan of having a var in the translation key.